docs/platform/concepts/authentication-tokens.rst

```diff
@@ -0,0 +1,35 @@
+Authentication tokens
+=====================
+
+An authentication token allows a user to programmatically access resources instead of using their username-password. Unlike the username-password combination that has a longer shelf life (in some case, they never expire); you can limit how long an authentication token is valid for or revoke it completely.
+
+.. mermaid::
+
+sequenceDiagram
```

Comment:
This generates an error in the preview 
![image](https://user-images.githubusercontent.com/11272743/151139569-8c31b661-5622-4b61-8124-5681a1e971f2.png)

Reply:
Yeah this is the main reason I had the PR in draft. Not sure on the change to render the valid diagram.

Reply:
Fixed, you needed an extra <TAB>